assets/rpip-62/rewards-calculation-spec.md

```diff
@@ -7,12 +7,12 @@ This document serves as a formal specification for the way that the rewards inte
 
 This describes **v10** of the rewards calculation ruleset.
 
-This version implements RPIP-[to be assigned]
+This version implements [RPIP-62](../../RPIPs/RPIP-62.md).
```

Comment:
"Implements the reward calculation component of RPIP-62"? Or something?

Reply:
sure